packages/acceptit-Core.package/String.extension/instance/withLeadingAuxiliaryNormalized.st

```diff
@@ -1,15 +0,0 @@
-*acceptit-Core
-withLeadingAuxiliaryNormalized
-
-	({
-		{'am'.'are'.'is'} -> 'is'.
-		{'have'.'has'} -> 'has'.
-		{'shall'. 'should'} -> 'should'.
-		{'may'. 'might'} -> 'may'.
-		{'will'. 'would'} -> 'will'.
-		{'can'.'could'} -> 'can'.
-	} as: Dictionary) keysAndValuesDo: [:key :value|
-		(self beginsWithAnyOf: key) 
-			ifTrue:[^ value, String space, (self copyAfter: Character space)]].
-		
-	 ^ self
```

Comment:
the general idea of normalizing scenarios(this is for scenario steps right?) seems not to bad so you could use an existing step implementation again although you have minor grammer changes in the new scenario, but actually kind of useless and unintuitive